articles/install-guide/index.md

```diff
@@ -17,237 +17,11 @@ Learn how to install the Microsoft Quantum Development Kit (QDK), so that you ca
-   - Select **Q#: Install project templates**
-
-    You now have the Quantum Development Kit installed and ready to use in your own applications and libraries.
-
-1. Verify the installation by creating a `Hello World` application
-
-    - Create a new project:
-
-        - Go to **View** -> **Command Palette**
-        - Select **Q#: Create New Project**
-        - Navigate to the location on the file system where you would like to create the application
-        - Click on the **Open new project...** button, once the project has been created
-
-    - Run the application:
-
-        - Go to **Debug** -> **Start Without Debugging**
-        - You should see the following text in the output window `Hello quantum world!`
-
-> [!NOTE]
-> * > * Workspaces with multiple root folders are not currently supported by the Visual Studio Code extension. If you have multiple projects within one VS Code workspace, all projects need to be contained within the same root folder.
-
-## Develop with C#, using the `dotnet` command-line tool
-
-Of course, you can also build and run Q# programs from the command line by simply installing the .NET Core SDK and the QDK project templates. 
-
-1. Pre-requisites
-
-    - [.NET Core SDK 3.0 or later](https://www.microsoft.com/net/download)
-
-1. Install the Quantum project templates for .NET
-
-    ```bash
-    dotnet new -i Microsoft.Quantum.ProjectTemplates
-    ```
-
-    You now have the Quantum Development Kit installed and ready to use in your own applications and libraries.
-
-1. Verify the installation by creating a `Hello World` application
-
-    - Create a new application
-
-       ```bash
-       dotnet new console -lang Q# -o runSayHello
-       ```
-
-    - Navigate to the new application directory
-
-       ```bash
-       cd runSayHello
-       ```
-
-    You should see that two files have been created, along with the project files of the application: a Q# file (`Operation.qs`) and a C# host file (`Driver.cs`).
-
-    - Run the application
-
-        ```bash
-        dotnet run
-        ```
-
-        You should see the following output: `Hello quantum world!`
-
-## What's next?
-
-Now that you have installed the Quantum Development Kit in your preferred environment, you can write and run [your first quantum program](xref:microsoft.quantum.write-program).
+- [Q# + Python:](xref:microsoft.quantum.install.python) choose this environment if you want to combine Python and Q# to create a Python host program that calls Q# operations whenever they are needed.
+- [Q# Jupyter Notebooks:](xref:microsoft.quantum.install.jupyter) this environment is great for writing Q# code with embedded explanations or quantum computing interactive tutorials. Do not choose this environment if you want to combine Q# with a classical host program.
+- [Q# + C#:](xref:microsoft.quantum.install.cs) choose this environment if you want to combine C# and Q# to create hybrid quantum-classical programs.
```

Comment:
- I'd suggest re-ordering these to have Jupyter Notebooks last, seeing as it's the odd one out
- also, while I support keeping the writing fresh and not repetitive, in this situation I think the Python and C# snippets should be nearly identical (or just tell them to pick the host language they're more comfortable with, because both can do the same things). As it is here, I'd be panicking wondering "wait, if i pick Python, will I be unable to 'create hybrid quantum-classical programs' like I could with C#?"

Reply:
I agree with the first point (I was thinking to change it but I forgot lol). With the second one I'm not sure. If I worded differently is because, as I understood, there are some things you can do with C# that you cannot do with Python yet. Maybe @cgranade has something to say here.

Reply:
It is true that we have some missing features with respect to the Python host functionality, but both C# and Python can definitely be used to create "hybrid quantum–classical programs." (In fact, Q# on its own can include Turing-complete classical logic without qubit allocation, so it might be good to clarify between classical logic in host programs and classical logic in Q# programs a little bit regardless.)